utest/test_robotlibcore.py

```diff
@@ -161,3 +165,19 @@ class OldStyle:
         HybridCore([OldStyle()])
     assert str(exc_info.value) == \
            "Libraries must be modules or new-style class instances, got old-style class 'OldStyle' instead."
+
+
+@pytest.mark.skipif(sys.version_info[0] == 2, reason='Only applicable on Py 3')
+@pytest.mark.skipif(robot__version >= '3.2', reason='For RF 3.1')
```

Comment:
As already commented earlier, `**kwargs` works with Python 2 just fine. It also works also with earlier Robot Framework versions. I'm also slightly surprised if there weren't tests for it already.

Reply:
There was a test that had `*varargs` and `**kwargs` but not test for kwargs alone.